src/lib.rs

```diff
@@ -333,6 +347,27 @@ mod tests {
     }
 
     #[test]
+    fn iter_random_add() {
+        use std::num::Wrapping as Z;
+        let mut set = BitSet::new();
+        let mut state = Z(0u64);
+        let max_added = 1_048_576 / 10;
+        let mut added = 0;
+        for _ in 0..max_added {
+            let oldstate = state;
+            state = oldstate * Z(6364136223846793005) + Z(1);
```

Comment:
Yeah, should probably do that. I didn't want to include rand at first just for tests, but if they are dev-dependency it should be fine.

(That magic is [PCG](http://www.pcg-random.org/) random number generator)